README.md

```diff
@@ -153,6 +154,8 @@ options:
   --lat LAT      center the map at this latitude, default is mean of all points
   --lon LON      center the map at this longitude, default is mean of all points
   --zoom ZOOM    initial zoom, default is 18
+  --types TYPES  comma separated list of message types, the default is GPS_RAW_INT and GPS_GLOBAL_INT
+  --hdop HDOP    reject GPS_INPUT messages where hdop exceeds this limit, default 100.0 (no limit)
```

Comment:
Would this flag be more intuitive as --hdop_max?

Reply:
Good call